i18n/en/docusaurus-plugin-content-docs/current/config/dev-server.mdx

```diff
@@ -0,0 +1,143 @@
+import DevServerExample from "@site/docs/config/_partials/examples/_dev-server-example.mdx";
+
+# devServer
+
+With the `devServer` section, you can run a server that tests will interact with. The server starts during the initialization of Testplane on the INIT event.
+
+## Example Usage {#example}
+
+<DevServerExample />
+
+## devServer Section Reference
+
+<table>
+    <thead>
+    <tr>
+        <td>**Parameter**</td>
+        <td>**Type**</td>
+        <td>**Default**</td>
+        <td>**Description**</td>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td>[`command`](#command)</td>
+        <td>`string`</td>
+        <td>_N/A_</td>
+        <td>
+            Command to start the dev server. If not specified, the dev server will not be started.
+        </td>
+    </tr>
+    <tr>
+        <td>[`env`](#env)</td>
+        <td>`Record<string, string>`</td>
+        <td>`{}`</td>
+        <td>
+            Environment variables to be passed to the dev server process, in addition to the main process's `process.env`.
+        </td>
+    </tr>
+    <tr>
+        <td>[`args`](#args)</td>
+        <td>`string[]`</td>
+        <td>`[]`</td>
+        <td>
+            Arguments to be passed to the dev server process.
+        </td>
+    </tr>
+    <tr>
+        <td>[`cwd`](#cwd)</td>
+        <td>`string`</td>
+        <td>_The nearest directory with package.json relative to the Testplane config_</td>
+        <td>
+            Working directory of the dev server.
+        </td>
+    </tr>
+    <tr>
+        <td>[`logs`](#logs)</td>
+        <td>`boolean`</td>
+        <td>`true`</td>
+        <td>
+            Enables streaming of dev server logs to the console with the prefix `[dev server]`.
+        </td>
+    </tr>
+    <tr>
+        <td>[`readinessProbe`](#readiness-probe)</td>
+        <td>`(devServer: ChildProcess) => Promise<void/> | ReadinessProbeOpts`</td>
+        <td>`null`</td>
+        <td>
+            Check that the server is ready to accept requests. If not specified, the check is disabled.
+        </td>
+    </tr>
+    </tbody>
+</table>
+
+### command {#command}
```

Comment:
Why you need to specify `{#command}`? Shouldn't the exact same value be used by default from title?

Reply:
Yeah it's command by default, but why not specify it everywhere to look consistent and guarantee that anchors won't break if the heading changes